test/uk/gov/hmrc/frontendtemplateprovider/FooterSpec.scala

```diff
@@ -36,17 +37,16 @@ class FooterSpec extends WordSpec with Matchers  with Results with WithFakeAppli
 
   implicit val hc = HeaderCarrier()
 
+  val fakeRequest = FakeRequest("GET", "/")
+
   override lazy val fakeApplication: Application = new GuiceApplicationBuilder()
-    .configure(Map("assets.url" -> "www.example.com/"))
-    .bindings(bindModules:_*)
+    .configure(Map("assets.url" -> "www.example.com/", "assets.version" -> "1"))
```

Comment:
Is this over-riding the values being supplied by the application.conf file? It did seem there was some value in testing that the values from the configuration file were being applied correctly to the served template.